docs/AdvancedFeatures.md

```diff
@@ -215,19 +218,21 @@ There is also a multi-line variant of the DEF statement that allows to define co
       - name: "Value ${simpleMap[loopVar]}"
 ```
 
+### Global variables 
+
 By default, variables are local, this means the scope of a variable is limited to:
  * the lines in the very same yaml file following the definition till it is either redefined or the end of the yaml file is reached 
  * `FOR` loop in which variable is defined or re-defined.
```

Comment:
Please add a sentence about precedence of variables (I guess same name local variable takes precedence) and also add tests for same name local and global variables

Reply:
See 1fec561